[Cookies] Increased cookies banner font size for med/large screens to improve readability

luisramos0 opened the pull request:
#### What? Why?

Closes #2597 

Increased font size for med/large screens and adapted CSS to keep the text and button visible in all screen sizes.

#### What should we test?
The cookies banner should be tested in terms of UX in different screen sizes. Functionality was not changed.

#### Release notes
Changelog Category: Changed

Cookies banner font size increased.

luisramos0 commented:
I am not sure but I dont think the broken build is related to this PR...

the error is:
Using embedded shopfront functionality using iframes allows shopping and checkout
     Failure/Error: expect(page).to have_text 'Your shopping cart'
       expected to find text "Your shopping cart" in "Powered by Open Food Network English Castellano English Login 1 items My Embedded Hub Herndon Next order closing in 2 days Ready for ABOUT MY EMBEDDED HUB PRODUCERS CONTACT GROUPS Ruby on Rails Framed Apples from Enterprise 333 Framed Apples 1g, S $19.99 $19.99". (However, it was found 2 times including non-visible text.)
     # ./spec/features/consumer/shopping/embedded_shopfronts_spec.rb:53:in `block (4 levels) in <top (required)>'

mkllnk commented:
I just ran Semaphore again and the same spec failed again. I think it needs fixing even though it may be unrelated to this PR.

luisramos0 commented:
I rerun the build and its green now.

luisramos0 commented:
Yes Matt!  I need to leave global config as it was before the test. Which is banner disabled. That is probably causing the error. I'll do that and add a commit to this PR. Also, Ill put that (must keep global state in your tests) to the code conventions.
Thanks!

Matt-Yorkley commented:
Great! I've seen other random failing feature specs this week for "missing" UI elements which then passed after a restart, so I'm guessing it's the same thing.

Matt-Yorkley commented:
P.S. @luisramos0 for debugging issues like this locally you can add the line `save_and_open_screenshot` temporarily to a spec at the point where the issue is, and it will give you a screenshot of what capybara is seeing.

luisramos0 commented:
Thanks @Matt-Yorkley 
I didnt know about that trick to get a screenshot, it's great! I added it here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Testing-and-Rspec-Tips#save_and_open_screenshot
I have a little local script that switches to selenium :-)

The convention "leave config as it was before the test" was already here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Testing-and-Rspec-Tips#clearing-app-config-changes-from-cache

I have added a new commit to fix it for the cookies_spec.

Thanks guys!

Matt-Yorkley commented on spec/features/consumer/cookies_spec.rb:
```diff
@@ -52,6 +60,14 @@
+
+    # keeps config unchanged
+    around do |example|
+      original_config_value = Spree::Config[:cookies_policy_matomo_section]
```
I see you've added this for these 2 specs, but is there a global config to disable it on all the other specs by default?

Matt-Yorkley replied:
Ah ok, I wasn't sure about the defaults.

luisramos0 commented:
I just staged this one here: https://staging.openfoodfrance.org/
I also went to the backoffice and activated the banner to have a look myself.

myriamboure commented:
I used Mozilla "test views" developer tool, pretty useful to check how it looks like on various screen sizes ;-) It doesn't deserve testing note, just a screen share:
It is more readable and decent in term of appearance, so good to go.
![capture du 2018-09-03 13-15-45](https://user-images.githubusercontent.com/8135376/44984021-93c80e80-af7b-11e8-85ae-653ac2dd928c.png)
I checked also on mobile as well of course, all good.
